protocol/wasmbinding/msg_plugin.go

```diff
@@ -129,6 +149,43 @@ func (m *CustomMessenger) depositToSubaccount(ctx sdk.Context, contractAddr sdk.
 	return nil, nil, nil
 }
 
+func (m *CustomMessenger) withdrawFromSubaccount(ctx sdk.Context, contractAddr sdk.AccAddress, withdrawFromSubaccount *bindings.WithdrawFromSubaccount) ([]sdk.Event, [][]byte, error) {
```

Comment:
`contractAddr` is passed in but not used.

Reply:
I agree it should just be for 3. Should be easy enough to address